src/main/java/com/CoachPlan/dto/CoachDTO.java

```diff
@@ -3,13 +3,26 @@
 public class CoachDTO {
 	
 	private String loginID;
+	private String fullName;
+	private String Sport;
```

Comment:
Attributes/variables should be lowerCamelCase in Java.  Make this sport, not Sport.